### DISCUSSION TITLE
Can we create  style all Antd components with Tailwindcss ?

### DISCUSSION THREAD

**Discussion by videni** [ant-design/ant-design#33724] [General]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
Tailwindcss is truely a life saver, it would be great if we can style Antd with Taildwindcss. which may need tremendous work, but it is worth to do.

### What does the proposed API look like?
None

**Comment by MadCcc** [comment]
What you want is that can overriding Antd styles with Tailwindcss classes?
Since we use some nested classes to make style work properly, I think it's hard to make it...

**Reply by Rainnut** [reply]
how can i do? i need to override antd by tailwindcss

**Comment by videni** [comment]
@MadCcc ,  not just override, instead , unleash the ability to style Antd components with any CSS tool, SASS, LESS , Tailwindcss.   the  [headlessui ](https://headlessui.dev/)  is very interesting,  the ui component itself has no styles at all. it would be awesome, if Antd  follows this approach.

**Comment by hokagedemehin** [comment]
This will make developers lean on this UI library if they can pull this off. It is not really about the nested classes available from Antd, but that little freedom to customize components based on how you want them to look. Although it is possible to use inline styles to customize some of the components, integrating CSS-in-js into the mix will change the gane
